contrib/deploy.sh

```diff
@@ -1,3 +1,19 @@
 #!/bin/bash
 
 export KUBECONFIG="$(kind get kubeconfig-path)"
+
+echo "Applying namespaces"
+kubectl apply -f ./namespaces.yml
+
+PASSWORD="something_random"
+kubectl -n openfaas create secret generic basic-auth \
+--from-literal=basic-auth-user=admin \
+--from-literal=basic-auth-password="$PASSWORD"
+
+helm install ./chart/openfaas \
```

Comment:
This is not how install works, you don't need to generate a file or apply it with kubectl. Please use `install --wait`

Reply:
Did `install --wait`